nose-locust/requirements.txt

```diff
@@ -6,3 +6,4 @@ pyzmq==14.6.0
 requests[security]==2.6.0
 fabric==1.10.1
 flaky==1.1.0
+docker-py>=1.3.0
```

Comment:
I think it might make more sense to specify the exact version of docker-py to use , as this repo is set up to autobuild on Docker Hub (bmangold/nose-locust).  If we need docker-py 1.4.0 for some reason, we would have to go and manually trigger an automated build for this image and then if there are incompatibilities with 1.4 and 1.3 , its not clear to the user which exact version is being used in bmangold/nose-locust.